tests/test_cfg_end_to_end_tests.py

```diff
@@ -188,14 +215,16 @@ def test_configs(config_path: str, data_check_fn: Callable, tmp_path: Path):
         generated_lines = sorted(generated_fin.readlines())
         assert len(reference_lines) == len(generated_lines)
 
-        for reference_line, generated_line in zip(reference_lines, generated_lines):
-            reference_data = json.loads(reference_line)
-            generated_data = json.loads(generated_line)
-            reference_data.pop("audio_filepath", None)
-            generated_data.pop("audio_filepath", None)
-            assert reference_data == generated_data
+        # if any non-deterministic processors are used (e.g. inference with Whisper), 
+        # we do not test the content of the output
+        if not non_det_processors:
```

Comment:
Can we exclude just one nondeterministic field?

Reply:
done